configs/fedora-sig-i686-steam.yaml

```diff
@@ -0,0 +1,46 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Packages Required By Steam Spec File
+  description: Packages Required By Steam
+  maintainer: ssmoogen
+  packages:
+    # The minimum development environment is this:
+    - desktop-file-utils
+    - libappstream-glib
+    - make
+    - systemd
+    - tar
+    - zenity
```

Comment:
These are Requires, not BuildRequires, but are programs and therefore the x86_64 builds would suffice.

Reply:
+1 

But also we should assume that there will be a lot more tweaking and adjustments once we get the initial workload definition publicly visible. Thus I would not be too picky for the items on the current list.

I think that the first step is to get _some_ definition of the workload in, publish it, and then invite people to scrutinize it further.